### PR TITLE
Remove expiry variable

### PR DESCRIPTION
**Pull request by mrice32**
Expiry variable doesn't need to be stored since we already have liquidation time.

### CODE REVIEW

**Comment by mrice32** [comment]
@chrismaree could you take a look at this PR?

File: core/contracts/financial-templates/implementation/Liquidatable.sol [hunk 1]
```diff
@@ -420,4 +420,8 @@ contract Liquidatable is PricelessPositionManager {
         require(uuid < liquidationArray.length && liquidationArray[uuid].state != Status.Uninitialized);
         return liquidationArray[uuid];
     }
+
+    function _getLiquidationExpiry(LiquidationData storage liquidation) internal view returns (uint) {
```
Comment (chrismaree) [inline core/contracts/financial-templates/implementation/Liquidatable.sol]: This is good. We don't need to calculate the expiry up front if it's always a function of `liquidationTime` and the `liquidationLiveness`

File: core/contracts/financial-templates/implementation/Liquidatable.sol [hunk 2]
```diff
@@ -30,19 +30,19 @@ contract Liquidatable is PricelessPositionManager {
...
         Status state; // Liquidated (and expired or not), Pending a Dispute, or Dispute has resolved
+        uint liquidationTime; // Time when liquidation is initiated, needed to get price from Oracle
         /** Following variables determined by the position that is being liquidated */
         FixedPoint.Unsigned tokensOutstanding; // Synthetic Tokens required to be burned by liquidator to initiate dispute
         FixedPoint.Unsigned rawLockedCollateral; // Collateral locked by contract and released upon expiry or post-dispute
         // Amount of collateral being liquidated, which could be different from
         // lockedCollateral if there were pending withdrawals at the time of liquidation
         FixedPoint.Unsigned liquidatedCollateral;
-        /** Following variables set upon a dispute request */
+        /* Set upon initiation of a dispute */
```
Comment (chrismaree) [inline core/contracts/financial-templates/implementation/Liquidatable.sol]: nit: this styling is inconsistent with the other section breaks within this struct. Should be:
```
/** Set upon initiation of a dispute */
```

Comment (mrice32) [reply core/contracts/financial-templates/implementation/Liquidatable.sol]: Done.

**Pull request merged**